src/plugins/controls/public/options_list/components/options_list_popover_footer.tsx

```diff
@@ -41,19 +47,32 @@ export const OptionsListPopoverFooter = () => {
   return (
     <>
       <EuiPopoverFooter
-        paddingSize="s"
+        paddingSize="none"
         css={css`
           background-color: ${useEuiBackgroundColor('subdued')};
         `}
       >
-        <EuiButtonGroup
-          legend={OptionsListStrings.popover.getIncludeExcludeLegend()}
-          options={aggregationToggleButtons}
-          idSelected={exclude ? 'optionsList__excludeResults' : 'optionsList__includeResults'}
-          onChange={(optionId) => dispatch(setExclude(optionId === 'optionsList__excludeResults'))}
-          buttonSize="compressed"
-          data-test-subj="optionsList__includeExcludeButtonGroup"
-        />
+        {isLoading && (
+          <div style={{ position: 'absolute', width: '100%' }}>
+            <EuiProgress size="xs" color="accent" />
+          </div>
+        )}
```

Comment:
The new loading state looks like this:

![Feb-02-2023 13-58-10](https://user-images.githubusercontent.com/8698078/216448038-308f87ff-cf43-4f01-b064-de8d1949a386.gif)

Note that I had to throttle my network to make this happen - because the loading is debounced, the loading state won't even appear if the server responds quick enough.